src/main/webapp/style/components/_button-1-of-2.css

```diff
@@ -0,0 +1,35 @@
+/*
+  Copyright 2020 Google LLC
+
+  Licensed under the Apache License, Version 2.0 (the "License");
+  you may not use this file except in compliance with the License.
+  You may obtain a copy of the License at
+
+      https://www.apache.org/licenses/LICENSE-2.0
+
+  Unless required by applicable law or agreed to in writing, software
+  distributed under the License is distributed on an "AS IS" BASIS,
+  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+  See the License for the specific language governing permissions and
+  limitations under the License.
+*/
+
+/*
+  Uses fractional naming (i.e. a button that takes up half the available space)
+ */
+.button-1-of-2 {
+  align-items: center;
+  border-radius: 1000px;
```

Comment:
Nit: Would it make sense to use a variable instead of a hard-coded value?

Reply:
Yep, I can change this up. Thanks!